invenio_vocabularies/contrib/affiliations/config.py

```diff
@@ -42,12 +42,9 @@ class AffiliationsSearchOptions(SearchOptions):
             # Aliases can sometimes be shorter, so we boost them a bit.
             "aliases^5",
             localized_title,
-            "id^2",
```

Comment:
I think this is going to break searching by ROR. The test data might be misleading here…in real data the id isn’t the same as the acronym.

Reply:
Thanks! I've added a test, it works